### PR TITLE
CSS for discount form

### PR DESCRIPTION
**Pull request by espenmn**

### CODE REVIEW

File: src/bda/plone/discount/browser/discount.css [hunk 1]
```diff
@@ -90,4 +90,8 @@ div.array_field.valid_to .timepicker {
 .array_field.threshold,
 .array_field.value {
     padding: 0 0.25em;
-}
\ No newline at end of file
+}
+
+.array_field {
+display: table-cell
```
Comment (rnixx) [inline src/bda/plone/discount/browser/discount.css]: Can you indent the rule by 4 spaces please?

Comment (thet) [inline src/bda/plone/discount/browser/discount.css]: also a semicolon is missing at the end of this line.

**Comment by thet** [comment]
can you please make a statement what your pull request fixes? i have to few context to know if it's good to merge or not.

**Comment by rnixx** [comment]
please check latest master and reopen if necessary

**Pull request closed without merging**